pyproject.toml

```diff
@@ -17,7 +17,7 @@ packages = [{include = "csm_api_client"}]
 python = "^3.9"
 boto3 = "^1.21.0"
 inflect = ">=0.2.5"
-kubernetes = ">=22.0.0"
+kubernetes = "^24.0.0"
```

Comment:
Given that the previous restriction was just ">=22.0.0", it is probably not necessary to update this library at all, unless it's doing anything that requires anything specific to version 24 of the `kubernetes` package. It should be pretty harmless to make this change, but note that it will make this library incompatible with older builds of SAT (e.g. for CSM 1.5) that require an earlier Kubernetes version (e.g. version 22.y.z corresponding to Kubernetes 1.21). Builds would fail though, so it would be obvious if we tried to do that.

Reply:
Agreed, will probably close it without pushing this change, until its required.